examples/io/read_impedances_from_ant_neuro.py

```diff
@@ -0,0 +1,65 @@
+"""
+.. _ex-io-ant-impedances:
+
+======================================
+Getting impedances from ANT Neuro .cnt
+======================================
+
+The ``.cnt`` file format from ANT Neuro stores impedance information in the form of
+triggers. The function :func:`mne.io.read_raw_ant` reads this information and mark the
+time-segment during which an impedance measurement was performed as
+:class:`~mne.Annotations` with the description set in the argument
+``impedance_annotation``. However, it doesn't extract the impedance values themselves.
+To do so, use the function ``antio.parser.read_triggers``.
+"""
+
+from antio import read_cnt
+from antio.parser import read_triggers
+from matplotlib import pyplot as plt
+
+from mne.datasets import testing
```

Comment:
I saw another example using the testing dataset, is that OK or should I move this file elsewhere to run the example?

Reply:
Yes it's okay to use `testing` as a dataset in examples/tutorials :+1: